doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -953,35 +344,6 @@ cJSON
 Documentation
 =============
 
-* Added:
-
-  * Software maturity entries for security features: TF-M, PSA crypto, Immutable bootloader, HW unique key.
-  * Documentation for :ref:`ug_nrf70`.
-  * Documentation for the :ref:`lib_flash_map_pm` library.
-  * Documentation for the :ref:`lib_adp536x` library.
-  * Documentation for the :ref:`lib_flash_patch` library.
-  * :ref:`gs_debugging` section on the :ref:`gs_testing`.
-    Also added links to this section in different areas of documentation.
-  * :ref:`ug_thread_prebuilt_libs` as a separate page instead of being part of :ref:`ug_thread_configuring`.
-  * A section about NFC in the :ref:`app_memory` page.
-  * A section about :ref:`modem_trace_backend_uart_custom_board` in the :ref:`nrf_modem_lib_readme` page.
-  * A note in the :ref:`ug_ble_controller` about the usage of the Zephyr LE Controller.
-  * A section about :ref:`modem_trace` in the :ref:`ug_nrf91_features` page.
-  * A section about Gazell in the :ref:`app_memory` page.
-
-* Updated:
-
-  * :ref:`gs_recommended_versions` page with a new section about :ref:`gs_supported_OS`.
-  * :ref:`ug_nrf91` with the following changes:
-
-    * In the :ref:`ug_nrf9160_gs` guide, :ref:`nrf9160_gs_updating_fw_modem` section is now moved before :ref:`nrf9160_gs_updating_fw_application` because updating modem firmware erases application firmware.
-    * In the :ref:`ug_nrf9160` guide, the :ref:`build_pgm_nrf9160` section now mentions |VSC| and command line instructions.
-    * In the :ref:`ug_thingy91_gsg` guide, :ref:`programming_thingy` and :ref:`connect_nRF_cloud` sections now have different structure.
-    * The instructions and images in the :ref:`ug_thingy91_gsg` and :ref:`ug_nrf9160_gs` documentation pages now also mention accepting :term:`eUICC Identifier (EID)` when activating your iBasis SIM card from the `nRF Cloud`_ website.
-
-  * :ref:`ug_thread_configuring` page to better indicate what is required and what is optional.
-    Also added further clarifications to the page to make everything clearer.
-  * :ref:`ug_matter_tools` page with a new section about the ZAP tool.
-  * Updated TF-M/SPM related files, included in samples and applications with _ns build targets, to no longer include SPM as an alternative to TF-M.
+* |no_changes_yet_note|
```

Comment:
```suggestion
* Updated:

  *  :ref:`mcuboot:readme-ncs.rst` now includes information regarding the bootloader user guides in the |NCS| documentation.

* Removed:

  * References to the Zephyr Test Plan 
```

If the ref to the mcuboot docset doesn't work (it might not be implemented), this is the direct link to that page:  https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/mcuboot/readme-ncs.html